app.js

```diff
@@ -97,6 +98,7 @@ window.onload = async () => {
             return;
         }
         !firebase.apps.length ? firebase.initializeApp(localDevFirebaseConfig) : firebase.app();
+        window.DD_RUM && window.DD_RUM.init({ ...datadogConfig, env: 'dev' });
```

Comment:
Local dev has lots of temporary errors during testing, and we better not send these errors to DataDog.
```suggestion
```